backend/app/services/work_queue_service.py

```
@@ -1488,6 +1488,138 @@ def retry_message(self, user_id: int, message_id: int) -> QueueMessageResponse:
+            ):
+                raise ConflictException(
+                    f"Message is already in status {message.status}"
+                )
+
+            # Load the queue to determine processing mode
+            work_queue = (
+                db.query(Kind)
+                .filter(
+                    Kind.id == message.queue_id,
+                    Kind.kind == "WorkQueue",
+                    Kind.is_active == True,
```

Ruff (0.15.9)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)